.gitmodules

```diff
@@ -4,18 +4,12 @@
 [submodule "sw/firesim-software"]
 	path = sw/firesim-software
 	url = https://github.com/firesim/firesim-software
-[submodule "sim/firrtl"]
-	path = sim/firrtl
-	url = https://github.com/ucb-bar/firrtl
 [submodule "sim/midas"]
 	path = sim/midas
 	url = https://github.com/ucb-bar/midas
-[submodule "sim/barstools"]
-	path = sim/barstools
-	url = https://github.com/ucb-bar/barstools.git
 [submodule "sim/firechip"]
```

Comment:
This is a nitpick now... but if we stick with the FSim as top approach, we should rename this to match the new name of REBAR

Reply:
You're right, i was unsure how much to commit to the name.